addons/mail/static/src/core/web/chatter.js

```diff
@@ -239,9 +239,9 @@ export class Chatter extends Component {
         }
         const recipients = [...this.state.thread.recipients].slice(0, 5).map(({ partner }) => {
             const text = partner.email ? partner.emailWithoutDomain : partner.name;
-            return `<span class="text-muted" title="${escapeHTML(partner.email)}">${escapeHTML(
-                text
-            )}</span>`;
```

Comment:
Or the name in red or something else to indicate an issue?

Reply:
Hi @seb-odoo 
That's what I originally wanted, but @alexkuhn correctly pointed out that if there are multiple recipients, it would be confusing not to know for which recipient this "error" applies

Reply:
![1710494345180](https://github.com/odoo/odoo/assets/26395662/046aac9b-1fde-4a60-98cc-a3375e784bea)
For example, something like this, ( or in bald, even more strong than this )

Reply:
But it would be very easy to check if a recipient has no email if they enabled inbox or not before displaying the warning.

Reply:
But I don't think it's a real use case anyway, internal users that use inbox most definitely have an email set (used for login? for password reset? ...) people that don't want to warning to appear for them should just set an email.

Reply:
IMO I don't think we should necessarily put heavy style on items that have no email address.
Showing the email is really just another way to identify the recipients, and it just so happens that most people receive messages by email. However, if a recipient does not have any email address, this doesn't necessarily means this is an issue.

@paju-odoo What about underlying recipients that have an email address? Most email apps shows recipients that way in the "To:" field of composing a message, so copying that UI style would be familiar to most users:
<img width="795" alt="Screenshot 2024-03-15 at 10 57 59" src="https://github.com/odoo/odoo/assets/6569390/edcae535-bd71-4d74-9a3f-f73154ccdcf5">

Reply:
@zel-odoo I'm not a fan of the red
I think having [name] invalid email address is enough :)

Reply:
@alexkuhn 
I don't think that would be necessary. As @seb-odoo said, not having an email address doesn't necessarily means that there is an error, and that's also why I don't think we should go with a red or specifi design. However, displaying the info might be handy, which is why I prefer the "simple" solution: [name] invalid email address